### PR TITLE
MCC Number distance added to each MCC Electrical Load List output Exc…

### PR DESCRIPTION
**Pull request by Shrek271198**
…el file

### CODE REVIEW

**Comment by Shrek271198** [comment]
The MCC Number distances have been added with Dictionary as you adviced. Also I have followed the coding standards that you have created for us.

Thank you for the guidance.

File: src/eload.py [hunk 1]
```diff
@@ -230,9 +230,10 @@ class MechanicalEquipment:
     operation_mode: OperationMode
     rev: str
     procurement_rating: str
+    estimated_distance: int    # distance
```
Comment (DevanR) [inline src/eload.py:233]: I don't think there is a need for the comment here.

Could we change the name from "estimated_distance" to "estimated_distance_to_XXX". Where XXX is the thing at the other end?

 Why "int" can't the distances include decimals?

File: src/eload.py [hunk 2]
```diff
@@ -295,6 +296,7 @@ class MotorControlCenter:
     """Class for Motor Control Center details."""
 
     name: str
+    estimated_distance: int
```
Comment (DevanR) [inline src/eload.py:299]: Same comment here. Could you update "estimated_distance"?

File: src/eload.py [hunk 3]
```diff
@@ -520,14 +522,17 @@ class ClientMechanicalEquipmentList:
 
     mel: List[MechanicalEquipment] = field(default_factory=list)
     mcc_numbers: list = field(init=False)
+    mcc_numbers_dict: dict = field(init=False)
```
Comment (DevanR) [inline src/eload.py:525]: There is not need to append "_dict" to the variable here. Also, "mcc_numbers" is already used before. It should be something that includes distances?

File: src/eload.py [hunk 4]
```diff
@@ -630,15 +642,27 @@ def read_client_mel(mel):
...
+    return client_mel_builder(rows, mcc_numbers_dict)
 
 
 def els_builder(STANDARD, MEL):
     """ This method builds and returns the ELS dataclass object."""
 
     # Initialise Motor Control Centers
     MCC_List = []
-    for number in MEL.mcc_numbers:
+    for number, distance in MEL.mcc_numbers_dict.items():
```
Comment (DevanR) [inline src/eload.py:665]: Have a more descritive name that "distance".

**Pull request closed without merging**